src/SourceBuild/content/repo-projects/razor.proj

```diff
@@ -2,8 +2,8 @@
 
   <ItemGroup>
     <RepositoryReference Include="arcade" />
-    <RepositoryReference Include="runtime" />
```

Comment:
Is the removal of runtime correct for non-source build? I see razor has a product dependency on runtime: https://github.com/dotnet/razor/blob/667c05ce82ca55f15ecb4578274df4aa0af9d204/eng/Version.Details.xml#L4-L7

Reply:
yep, razor is a leaf.